### PR TITLE
Fixing podspec by removing dependency on patch version

### PR DESCRIPTION
**Pull request by pkrmf**
- ComponentKit will no longer have a dependency on the Yoga's patch version. We are going from 1.6.0 to 1.6.
- Bumped version of ComponentKit to 0.22.1. Let me know if that is OK. We should be looking to pushing this new podspec to cocoapods master repo

This PR solves [#913](https://github.com/facebook/componentkit/issues/913)

### CODE REVIEW

File: ComponentKit.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name = 'ComponentKit'
-  s.version = '0.22'
+  s.version = '0.22.1'
```
Comment (alickbass) [inline ComponentKit.podspec]: The new version will contain some breaking changes in the framework, so I would rather make it 0.23 instead

**Comment by alickbass** [comment]
Could you also add other changes like in [this PR](https://github.com/facebook/componentkit/pull/779/files)

File: ComponentKit.podspec (hunk 1 above)
Comment (pkrmf) [reply ComponentKit.podspec]: I would think Yoga 1.6.0 to 1.x shouldn’t have any breaking API changes. At the end, the third digit never involves api changes, it just for bug fixes. If that is not true in yoga’s case, then I can change the version to 0.23

Ideally, breaking API changes should only happen on major versions. In this scenario, if our dependency was moving from yoga 1.6 to 2.0

Comment (alickbass) [reply ComponentKit.podspec]: Yoga doesn't, but the ComponentKit does. So it's better to be safe

**Comment by pkrmf** [comment]
@alickbass I made the changes you requested

**Comment by pkrmf** [comment]
@alickbass is the CI responsible for tagging the repository and then publishing the podspec to the cocoapods master repo?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by alickbass** [comment]
I don't think so. I will have to do it manually, after this PR gets merged

**Pull request closed without merging**